### PR TITLE
Enda Yohannes further mss

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Enda Yohannes 14 - 20

### CODE REVIEW

**Review by eu-genia — APPROVED** [review]

File: EndaYohannesQollaGable/EYQG014.xml [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+                           <decoNote type="unspecified" xml:id="d2">
+                               <locus target="#2v"/>
+                               
+                               <desc>Red circles divided into segments by black lines, black and red decorative elements around (possibly an unfinished diagram).</desc>                            
+                           </decoNote>    
+                           
+                           <decoNote type="miniature" xml:id="d3">
+                               <locus target="#3v"/>
+                               <desc><ref type="authFile" corresp="AT1012Equestrian"/> <persName ref="PRS4585Georgeo">St George </persName>
+                                   on the hourse back, with a boy behind.</desc>
```
Comment (thea-m) [inline EndaYohannesQollaGable/EYQG014.xml:75]: on horseback

File: EndaYohannesQollaGable/EYQG014.xml [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+                           
+                           <decoNote type="miniature" xml:id="d3">
+                               <locus target="#3v"/>
+                               <desc><ref type="authFile" corresp="AT1012Equestrian"/> <persName ref="PRS4585Georgeo">St George </persName>
+                                   on the hourse back, with a boy behind.</desc>                            
+                           </decoNote> 
+                           
+                           <decoNote type="miniature" xml:id="d4">
+                               <locus target="#4r"/>
+                               <desc><ref type="authFile" corresp="AT1031VirginAndChild"/> (mirrored?) and two Engels.</desc>
```
Comment (thea-m) [inline EndaYohannesQollaGable/EYQG014.xml:80]: angels

File: EndaYohannesQollaGable/EYQG015.xml [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+                        </msIdentifier>
+                    <msContents>
+                       <summary/>
+                       <msItem xml:id="ms_i1">                        
+                           <title type="complete" ref="LIT2286ZenaSellase"/>
+                         <textLang mainLang="gez"/>
+                           <colophon xml:lang="am" xml:id="coloph1">
+                               <locus target="#120" cert="low"/>
+                               ይትባረክ፡ እግዚአብሔር፡ አምላከ፡ ፳ል፡ አነሳስቶ፡ ላስጀመረኝ፡ አስጀምሮ፡ ላስጨረሰኝ፡ ክብር፡ ምስጋና፡ ይግባው፡ 
+                               ዝተጽሕፈ፡ በ<date>፲፱፻፶፪ዓመተ፡ ምሕረት፡</date> በወንጀላዊ፡ <hi rend="rub">ዮሐንስ፡</hi> ወዘመነ፡ <persName ref="PRS5226haylase">ንጉሥነ፡ <hi rend="rub">ኃይለ፡ ሥላሴ፡</hi>
```
Comment (thea-m) [inline EndaYohannesQollaGable/EYQG015.xml:38]: hi rend="rubric" throughout

File: EndaYohannesQollaGable/EYQG017.xml [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+                   </physDesc>
+                     
+                  
+                    
+                      <history>
+                                                <origin>
+                                  <origDate notBefore="1969" notAfter="1970">In 1962 EC (1969/70 CE) or shortly before.</origDate>
+                                  Bought and donated by 
+                                  <foreign xml:lang="gez">wayzāro</foreign>
+                                   ʾAmāraš Gabra Mikāel, copied in or shortly before 1962 EC (1969/70 CE).
```
Comment (thea-m) [inline EndaYohannesQollaGable/EYQG017.xml:80]: Mikāʾel

**Review by thea-m — CHANGES_REQUESTED** [review]

File: EndaYohannesQollaGable/EYQG014.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply EndaYohannesQollaGable/EYQG014.xml:75]: ok

File: EndaYohannesQollaGable/EYQG014.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply EndaYohannesQollaGable/EYQG014.xml:80]: ok

File: EndaYohannesQollaGable/EYQG015.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply EndaYohannesQollaGable/EYQG015.xml:38]: indeed

File: EndaYohannesQollaGable/EYQG017.xml (hunk 4 above)
Comment (DenisNosnitsin1970) [reply EndaYohannesQollaGable/EYQG017.xml:80]: ok

**Pull request merged**